modules/brushingandlinking/include/modules/brushingandlinking/brushingandlinkingmanager.h

```diff
@@ -218,6 +219,19 @@ class IVW_MODULE_BRUSHINGANDLINKING_API BrushingAndLinkingManager : public Seria
      */
     void setParent(BrushingAndLinkingManager* parent);
 
+    /**
+     * Add a \p callback to the manager that gets called when a brushing action is triggered.
+     *
```

Comment:
I would not add so many empty lines...

Reply:
I wanted this to be a separate paragraph so it sticks out more.

Reply:
Probably better to use `\remark{}` https://www.doxygen.nl/manual/commands.html#cmdremark